CMakeLists.txt

```diff
@@ -238,8 +238,20 @@ if(NOT ARB_USE_BUNDLED_RANDOM123)
     target_include_directories(ext-random123 INTERFACE ${RANDOM123_INCLUDE_DIR})
 endif()
 
-add_subdirectory(ext)
+# in the event we can find hwloc, just add it
+find_package(HWLOC QUIET)
```

Comment:
It would be nice to have an option to control the inclusion of HWLOC. Especially for package managers like spack, where ideally it should fail if HWLOC is not found despite being requested.

Reply:
I can add an advanced option to force it, yes.

Reply:
Done.